native-packages/fs-search/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@parcel/fs-search",
+  "version": "2.0.0-beta.1",
+  "files": ["index.js", "*.node"],
+  "napi": {
```

Comment:
Shouldn't there be a browser entry that just does the same in JavaScript?

Reply:
This package is only used by NodeFS, which is probably not very useful in the browser anyway. And MemoryFS does have a JS fallback.

Reply:
Got it